### ISSUE TITLE
Staging Review finding: Button styled as a link

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99048]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 10-10 Health Apps
**Product:** 10-10CG
**Feature:** Improve Facility Selection

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/locator](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/apply-form-10-10cg/veteran-information/va-medical-center/locator) after completing a search, you get a `va-radio` group containing radio button tiles for your search results followed by a button labeled "Load more facilities." The button is styled to look like a link.

When using some assistive technologies (example: voice control / speech recognition), links and buttons have different interaction patterns. When a user who sees something that looks like a link, they may try to interact with it as a link but have that interaction fail because it's actually a button. Whenever possible, we try to maintain material honesty --- links look like links and navigate, buttons look like buttons and perform actions.

<details><summary>Link, screenshot or steps to recreate</summary>

![Radio tile group of VA medical facilities with a "Load more facilities" link at the bottom.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/51980-Brian-2.png?raw=true)

</details>

## Recommended action

Style the "Load more facilities" element to look like a button, eg. the secondary button style.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that the "Load more facilities" link has been changed to a button
<img width="943" alt="Image" src="https://github.com/user-attachments/assets/ad07d808-69e7-4ee6-8dff-ea081e2fd152" />
